### DISCUSSION TITLE
Request: Any Kubernetes support for the project?

### DISCUSSION THREAD

**Discussion by getkub** [fastapi/full-stack-fastapi-template#2136] [Questions] [answered]
We were looking for a "admin/control" stack to automate and make it easy for end-users and the full-stak-fastapi is perfect. But unfortunately our team have access to AKS/GKE (and no VM's). Is there an option to install this project in native Kubernetes? If yes, any documentation/notes which can replicate from docker swarm?

**Comment by tiangolo** [comment, marked as answer]
You can definitely deploy FastAPI to Kubernetes, this stack or any other, but sadly as Kubernetes is so, so complex, and so dependend on the specific installations and additions installed on top, there's no simple straightforward way to have generic configs that could apply to any Kubernetes.

I liked Docker Swarm mode as it's a lot simpler, but it lost the battle against Kubernetes...

Seeing this, and that I'm not happy with the state of products to handle deployments, I started building something that I would like to exist for FastAPI deployments, all this was a big part of the reason for that.
